weave-js/src/components/Panel2/ChildPanel.tsx

```diff
@@ -507,9 +507,7 @@ const useChildPanelCommon = (props: ChildPanelProps) => {
 };
 
 const varNameToTitle = (varName: string) => {
-  return varName
-    .replace(/_/g, ' ')
-    .replace(/\b\w/g, char => char.toUpperCase());
```

Comment:
just removed the g so it only capitalizes the first letter, so we get sentence case